Mathlib/Algebra/MvPolynomial/Basic.lean

```diff
@@ -920,6 +920,10 @@ theorem eval₂_C (a) : (C a).eval₂ f g = f a := by
 theorem eval₂_one : (1 : MvPolynomial σ R).eval₂ f g = 1 :=
   (eval₂_C _ _ _).trans f.map_one
 
+@[simp] theorem eval₂_ofNat (n : Nat) [n.AtLeastTwo] :
+    (no_index (OfNat.ofNat n) : MvPolynomial σ R).eval₂ f g = n :=
```

Comment:
Should it not have `OfNat.ofNat n` on the RHS?

Reply:
I think we're usually pretty sloppy about this; it only matters if someone tries to use the theorem with `simp` in reverse, and in this case it wouldn't work anyway because there's nothing to key on.

Reply:
I pushed a fix for this.